app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -3,9 +3,6 @@
 
   logo_link ||= "https://www.gov.uk/"
   logo_link_title ||= t("components.layout_super_navigation_header.logo_link_title")
-  logo_text = t("components.layout_super_navigation_header.logo_text")
```

Comment:
I don't think this is used anywhere anymore - this and the related translation keys can probably be removed

Reply:
Good spot, I've removed `logo_text` and any related translation keys in the latest commit